### PR TITLE
hello world update

### PR DESCRIPTION
**Pull request by adamkdean**
Fixed a few issues + added some missing sections.

### CODE REVIEW

File: _src/web/getting-started/hello-world.md [hunk 1]
```diff
@@ -97,32 +120,34 @@ Notice we are referencing the datasource we created by its assigned `key`.
...
+  "datasources": [
     "greeting"
-  },
-  "routes": [
-    {
-      "path": "/"
-    }
-  ]
+  ],
+  "route": {
```
Comment (jimlambie) [inline _src/web/getting-started/hello-world.md]: The original `routes` property is actually correct for Web 1.7+. See http://docs.dadi.tech/web/concepts/routing/#page-routing

Comment (abovebored) [inline _src/web/getting-started/hello-world.md]: cc @adamkdean

Comment (adamkdean) [inline _src/web/getting-started/hello-world.md]: Ack, my bad. I'll revise this.

**Comment by adamkdean** [comment]
@abovebored could you review this again please? Thanks 🙇

**Pull request merged**